resources/translations/en-GB.edn

```diff
@@ -95,6 +95,7 @@
                           :third-party-review "Reviewed by a third party"
                           :unknown "Unknown"
                           :withdraw "Withdrawn for modifying"}
+                 :request-review "Request review by a third party"
```

Comment:
Why is this needed? What about `:t.actions/review-request`?

Reply:
If that's not used elsewhere then it could be used instead and this one removed.